deno.yaml

```diff
@@ -1,7 +1,12 @@
 package:
   name: deno
+<<<<<<< HEAD
+  version: 1.43.6
+  epoch: 2
+=======
   version: 1.44.0
   epoch: 0
+>>>>>>> main
```

Comment:
@krishjainx hey, I believe by mistake you have leave the lines <<<<<<<< HEAD and all.

and as this PR is 2 weeks old there is a lot of package that have version update in the mean time.

One quick suggestion: If in future you have to do a downstream epoch bump like this, always start with update the branch with current main, there will be a lot less merge conflicts.